bin/build-languages.js

```diff
@@ -6,7 +6,7 @@ const languages = require( '@automattic/languages' );
 const parse = require( 'gettext-parser' ).po.parse;
 const _ = require( 'lodash' );
 const mkdirp = require( 'mkdirp' );
-const fetch = require( 'node-fetch' );
+const fetch = import( 'node-fetch' ); // Dynamic import because node-fetch is now ESM.
```

Comment:
Don't we need to `await` for it?

Reply:
@scinos, I did have to modify it to this, based on the docs:

```
 const fetch = ( ...args ) => import( 'node-fetch' ).then( ( { default: f } ) => f( ...args ) );
```

Since `fetch` is a promise-based function anyways, we're already awaiting it throughout the file

Reply:
Is it an option to convert this script to ESM and use `import` everywhere? Would `import fetch from 'node-fetch'` work then, or is there some additional gotcha? Jest never touches this file, does it?